docs/how-to-guides/how-to-add-dvc-to-a-service.md

```diff
@@ -6,7 +6,7 @@ This guide will help you add DVC to your service.
 
 ### Install DVC
 
-https://dvc.org/doc/install
+[https://dvc.org/doc/install](https://dvc.org/doc/install)

```

Comment:
Just so you know, you can write `<https://dvc.org/doc/install>` if you want to put a link without a description in Markdown.